From 1ae55ea85f32435ce85df0b02142e35058ea51b7 Mon Sep 17 00:00:00 2001
From: Tejas Khatiwala <tejas@cerebras.net>
Date: Wed, 24 Jan 2018 22:07:04 +0000
Subject: [PATCH] fixed "shift-negative_value" error

---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index dcbf3ef..3c7e88b 100644
--- a/Makefile
+++ b/Makefile
@@ -256,6 +256,7 @@ gdb_unzip:
 	# @if [ ! -f ${GDB}.tar.gz ] && [ -f /usr/bin/wget ]; then \
 	#  wget http://ftp.gnu.org/gnu/gdb/${GDB}.tar.gz; fi
 	@tar --exclude-from gdb.files -xvzmf ${GDB}.tar.gz
+	@sed -i 's/imm |= (-1 << 7);/imm -= 0x80;/g' ${GDB}/opcodes/arm-dis.c
 	@make --no-print-directory gdb_patch
 
 gdb_patch:
-- 
1.8.3.1

